src/main/java/com/cegeka/academy/domain/Invitation.java

```diff
@@ -11,6 +11,7 @@
 @Table(name = "invitation")
 @Cache(usage = CacheConcurrencyStrategy.NONSTRICT_READ_WRITE)
 public class Invitation {
+
```

Comment:
Poti sa faci override la equals si hashcode pt ca probabil vom tine instante din acest obiect in seturi/liste. https://www.geeksforgeeks.org/override-equalsobject-hashcode-method/